tests/xharness/TestImporter/Templates/Managed/Resources/src/common/TestRunner/Core/TcpTextWriter.cs

```diff
@@ -35,26 +35,25 @@ static string SelectHostName (string[] names, int port)
 			using (var evt = new ManualResetEvent (false)) {
 				for (int i = names.Length - 1; i >= 0; i--) {
 					var name = names [i];
-					ThreadPool.QueueUserWorkItem ((v) =>
-						{
-							try {
-								var client = new TcpClient (name, port);
-								using (var writer = new StreamWriter (client.GetStream ())) {
-									writer.WriteLine ("ping");
-								}
-								lock (lock_obj) {
-									if (result == null)
-										result = name;
-								}
-								evt.Set ();
-							} catch (Exception) {
-								lock (lock_obj) {
-									failures++;
-									if (failures == names.Length)
-										evt.Set ();
-								}
+					ThreadPool.QueueUserWorkItem ((v) => {
+						try {
+							var client = new TcpClient (name, port);
+							using (var writer = new StreamWriter (client.GetStream ())) {
+								writer.WriteLine ("ping");
+							}
+							lock (lock_obj) {
+								if (result == null)
```

Comment:
`if (result is null)` ?

Reply:
This PR only changes whitespace, to minimize the changes and make it easier to review (because you can ignore whitespace changes in the review: https://github.com/xamarin/xamarin-macios/pull/16857/files?w=1)